src/components/SecondaryPanes/BreakpointsDropdown.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Svg from "../shared/Svg";
+import Dropdown from "../shared/Dropdown";
+import "./BreakpointsDropdown.css";
+
+function debugBtn(type, tooltip) {
+  return (
+    <button className="button-spacer">
+      <Svg name={type} title={tooltip} aria-label={tooltip} />
+    </button>
+  );
+}
+
+function renderPause(pauseData, isWaitingOnBreak) {
+  if (pauseData) {
+    return debugBtn("resume", L10N.getFormatStr("resumeButton"));
+  }
```

Comment:
we don't need the resume button here... because we're just creating a new breakpoint